### ISSUE TITLE
Make `dmp_render()` take `content_type` and `status` args like Django `render()`

### ISSUE THREAD

**Issue by tgroshon** [doconix/django-mako-plus#20]
The Django render function allows the following args:

`render(request, template_name, context=None, content_type=None, status=None, using=None)`

Two very useful args are `content_type` and `status` which allow the caller to better tune the properties of the `HttpResponse` object that *DMP* will return.

Without this feature, the caller needs to do something like the following to set these values:

```python
html = dmp_render_to_string(request, 'template.html', params)
return HttpResponse(html, status=201, content_type='text/html')
```

**Comment by doconix** [comment]
Short version: Done.  Run `pip3 install --update django_mako_plus`.

Slightly longer version: I added these extra functions today.  I did not include using=None because that explicitly selects the template engine (and by running dmp_render you're already explicitly using DMP).  I added charset= as an extra parameter that I think Django should have added.

I tested it as thoroughly as I can.  Let me know if you hit any errors.
